neqo-http3/src/connection_client.rs

```diff
@@ -5946,26 +5946,11 @@ mod tests {
         }
     }
 
-    // Test that decoder stream type is always sent beofer any other instruction also
-    // in case when 0RTT is used.
-    #[test]
-    fn zerortt_request_use_dynamic_table() {
-        const MAX_TABLE_SIZE: u64 = 65536;
-        const MAX_BLOCKED_STREAMS: u16 = 5;
+    const MAX_TABLE_SIZE: u64 = 65536;
+    const MAX_BLOCKED_STREAMS: u16 = 5;
+
+    fn get_token(server: &mut Http3Server) -> ResumptionToken {
```

Comment:
```suggestion
    fn get_resumption_token(server: &mut Http3Server) -> ResumptionToken {
```

Probably best to say what sort.